index.js

```diff
@@ -2,8 +2,11 @@
 var objectAssign = require('object-assign');
```

Comment:
I'm ok on making the code to ES2015 parity but I did not want to do a bigger rewrite on a specific feature and also with my first ever contribution to this project, if @sindresorhus is ok I can do it.